<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>387</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="Login">
       <widget class="QPushButton" name="login">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>210</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="username">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>100</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="password">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>150</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>100</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>username</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>150</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Menu">
       <widget class="QPushButton" name="next">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>next</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>60</y>
          <width>301</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Menu 2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Personnel">
       <widget class="QLineEdit" name="Nom_edit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Adr_edit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Prenom_edit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Cin_edit">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>20</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Fct_edit">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>60</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>20</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cin</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>60</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fonction</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>100</y>
          <width>181</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>100</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date d'embauche</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>701</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="Ajouter_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>310</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Modifier_btn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>310</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Filtrer_btn">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>310</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Filtrer</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Supp_btn">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>310</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Salaire_edit_3">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>140</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>140</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Absense">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Raison</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_d'absence</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Etat_edit">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>30</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Raison_edit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>30</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>194</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>70</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Personnel</string>
        </property>
       </widget>
       <widget class="QComboBox" name="Personnel_cbx">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>70</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="Supp_btn_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>130</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Ajouter_btn_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>130</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Modifier_btn_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>130</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Filtrer_btn_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>130</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Filtrer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>701</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Salaire">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>140</y>
          <width>351</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Salaire 5</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Departement">
       <widget class="QToolButton" name="Supp_btn_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>701</width>
          <height>161</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="Ajouter_btn_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Libelle</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Modifier_btn_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QToolButton" name="Filtrer_btn_5">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Filtrer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Adress_dep_edit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="libelle_edit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_33">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Personnels</string>
        </property>
       </widget>
       <widget class="QComboBox" name="perso_combo">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>20</y>
          <width>171</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
